.github/workflows/ci.yml

```diff
@@ -176,6 +176,8 @@ jobs:
       - uses: ./.github/actions/upload
         with:
           test-report-retention-days: ${{ env.TEST_REPORT_RETENTION_DAYS }}
+          # only upload heap dumps if running on forks, where no secrets are used
+          upload-heap-dump: ${{ secrets.SECRETS_PRESENT != 'present' }}
```

Comment:
@findepi what's the value of this secret?

Reply:
I'd need to test for it to be empty. I'm not sure if it's possible to set an empty secret at all. It's enough if you confirmed that there is a non-empty value set.